tests/ui-testing/pages/logsPage.js

```
@@ -425,16 +422,16 @@ export class LogsPage {
         const date = new Date(dateText.trim());
         const startDate = new Date(startDateStr);
         const endDate = new Date(endDateStr);
-    
+
         // Check if the parsed date is valid
         if (isNaN(date.getTime())) {
```

Biome (1.9.4)

[error] isNaN is unsafe. It attempts a type coercion. Use Number.isNaN instead.

See the MDN documentation for more details.
Unsafe fix: Use Number.isNaN instead.


(lint/suspicious/noGlobalIsNan)